initializr-service/src/main/resources/application.yml

```diff
@@ -278,8 +278,11 @@ initializr:
           description: Persistence support using MyBatis
           groupId: org.mybatis.spring.boot
           artifactId: mybatis-spring-boot-starter
-          version: 1.1.1
-          versionRange: 1.3.0.RELEASE
+          mappings:
+            - versionRange: 1.4.0.M2
+              version: 1.2.0
+            - versionRange: "[1.3.0.RELEASE,1.4.0.M2)"
```

Comment:
The mybatis-spring-boot-starter 1.2.0 is used the `ObjectProvider` added by Spring 4.3. However, 1.4.0.M1 cannot use the `ObjectProvider`.